pymc/logprob/transforms.py

```diff
@@ -672,7 +672,10 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
 
     # Do not apply rewrite to discrete variables
     if measurable_input.type.dtype.startswith("int"):
-        return None
+        if check_negation(node.op.scalar_op, node.inputs[0]) is False and not isinstance(
```

Comment:
Let's also check the output is still integer (that is the added value or mul -1 where also integers)